pkg/controllers/provisioning/scheduling/topologygroup.go

```diff
@@ -188,14 +198,21 @@ func (t *TopologyGroup) domainMinCount(domains *scheduling.Requirement) int32 {
 	}
 
 	min := int32(math.MaxInt32)
+	var numPodDomainsWithNode int
 	// determine our current min count
 	for domain, count := range t.domains {
 		if domains.Has(domain) {
+			numPodDomainsWithNode++
```

Comment:
@jonathan-innis I admit I'm sort of naively guessing that this is where "number of podDomains with at least a single node in the domain" could be accounted for

Reply:
now that we simply evaluate based on the number of podDomains, do we want to continue w/ the above evaluation against the "number of pod domains that have a node"?

Reply:
removed